backend/src/core/config.py

```
@@ -48,6 +48,10 @@ class Settings(BaseSettings):
     OPENAI_API_KEY: Optional[str] = None
     OPENAI_MODEL: str = "gpt-4"
 
+    # File uploads
+    UPLOAD_DIR: str = "/tmp/career-scout-uploads"
```

Ruff (0.15.7)

[error] Probable insecure usage of temporary file or directory: "/tmp/career-scout-uploads"

(S108)